src/main/java/org/semantics/apigateway/api/SkosmosTransformer.java

```diff
@@ -0,0 +1,64 @@
+package org.semantics.apigateway.api;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class SkosmosTransformer implements DatabaseTransformer {
+    @Override
+    public Map<String, Object> transformItem(Map<String, Object> item) {
+        if (item == null) {
+            return null;
+        }
+
+        Map<String, Object> transformedItem = new HashMap<>();
+
+        // Check for null values before accessing properties
+        if (item.containsKey("@id") && item.get("@id") != null) {
+            transformedItem.put("iri", item.get("@id"));
```

Comment:
this line will never be run, as `item.get(@id) != null` will always be false. At the level f the transformers, the item object will always have this fields: `iri, label, synonym, short_form, description, ontology, type, source and backend_type`
![image](https://github.com/user-attachments/assets/5065e7c7-602f-4e6c-b1a1-97ba3499a861)

Reply:
code cleaning will be done in the next iterations, we will focus on Skosmos integration only on this PR.